go/vt/vtgate/planbuilder/testdata/unsupported_cases.txt

```diff
@@ -377,3 +377,7 @@
 # Multi shard query using into outfile s3
 "select * from user into outfile s3 'out_file_name'"
 "unsupported: this construct is not supported on sharded keyspace"
+
+# unsupported two predicates specifying the database for the same table
+"SELECT cc.constraint_name AS 'name' FROM information_schema.check_constraints cc  WHERE cc.constraint_schema = 'constraint_schema' AND cc.table_schema = 'a'"
```

Comment:
I'm guessing `WHERE cc.constraint_schema = 'foo' AND cc.table_schema = 'foo'` would also not be accepted
Do you think we can make the planner clever enough to check and accept the two predicates as long as they have the same value?

Reply:
Yes absolutely, I'll take a look

Reply:
done in the last commit